src/server/full-server.ts

```diff
@@ -0,0 +1,25 @@
+import "../moduleAlias";
+import {RootConfig} from "$common/config/RootConfig";
+import {RillDeveloper} from "$common/RillDeveloper";
+import {ExpressServer} from "./ExpressServer";
+import type {SocketNotificationService} from "$common/socket/SocketNotificationService";
+import {ServerConfig} from "$common/config/ServerConfig";
+
+/**
```

Comment:
is this comment still necessary at this point?

Reply:
Nope. The file itself is not needed anymore.